### PR TITLE
Help Hub - DQA fixes, Mobile menu fixes

### PR DESCRIPTION
**Pull request by redscar**
### 🗒️ Description
This fixes more DQA issues. @codingmusician helped me with the responsive styling of Help Hub.

I added in the mobile menu as well.

### 🎥 Artifacts 

[Help_Hub_Mobile_Styles.webm](https://github.com/user-attachments/assets/e46b60bb-9cfd-42be-ac6b-d1dde56081c1)

### CODE REVIEW

File: src/admin-views/help-hub.php [hunk 1]
```diff
@@ -24,22 +36,54 @@ class="tribe-events-admin-title__logo"
...
+			aria-controls="tec-settings-nav-modal"
+		>
+			<span><?php echo esc_html( $hub_tabs[0]['label'] ); ?></span>
+			<img
+				class="tec-modal__control-icon"
+				src="<?php echo esc_url( tribe_resource_url( 'images/icons/hamburger.svg', false, null, Tribe__Main::instance() ) ); ?>"
+				alt="<?php esc_attr_e( 'Open settings navigation', 'tribe-common' ); ?>"
+			>
+		</button>
+	</div>
```
Comment (rafsuntaskin) [inline src/admin-views/help-hub.php:71]: Please add some more aria attributes for the dialog and closing buttons like ` role="dialog"`

File: src/resources/postcss/help-hub.pcss [hunk 2]
```diff
@@ -32,47 +33,36 @@ ul.tec-help-list__list-expanded {
...
-.tec-settings__sidebar-icon-wrap {
-	display: grid;
-	grid-template-columns: 38px 1fr;
-}
-
-.tec-settings__sidebar-icon-wrap .tec-settings__sidebar-icon-wrap-content {
-	margin-left: 12px;
+.tribe_events_page_tec-events-settings .tec-settings-form {
+	width: auto;
+	padding-bottom: var(--tec-spacer-4);
```
Comment (codingmusician) [inline src/resources/postcss/help-hub.pcss:38]:
```suggestion
	padding-bottom: var(--tec-spacer-4);
	width: auto;
```

File: src/resources/postcss/help-hub.pcss [hunk 3]
```diff
@@ -32,47 +33,36 @@ ul.tec-help-list__list-expanded {
...
-	.tec-tab-container {
-		display: none;
-		visibility: hidden; /* Hide initially to avoid flickering */
-		flex-flow: row wrap;
-	}
+/* Sidebar Styles */
+.tribe_events_page_tec-events-settings .tec-settings-form__sidebar > .tec-settings-form__sidebar-section {
+	margin: 0 auto auto 0;
+	border: 1px solid var(--tec-color-border-default);
+	width: calc(100% - var(--tec-spacer-4) * 2);
```
Comment (codingmusician) [inline src/resources/postcss/help-hub.pcss:45]:
```suggestion
	border: 1px solid var(--tec-color-border-default);
	margin: 0 auto auto 0;
	width: calc(100% - var(--tec-spacer-4) * 2);
```

File: src/resources/postcss/help-hub.pcss [hunk 4]
```diff
@@ -32,47 +33,36 @@ ul.tec-help-list__list-expanded {
...
 
-/* Content Section Styling */
-.tec-settings-form__content-section {
-	padding-right: var(--tec-spacer-7);
-	width: calc(100% - var(--tec-spacer-3) * 2);
+.tribe_events_page_tec-events-settings .tec-settings-form__sidebar {
+	display: flex;
+	max-width: 100%;
+	width: 100%;
+	min-height: auto;
```
Comment (codingmusician) [inline src/resources/postcss/help-hub.pcss:52]:
```suggestion
	display: flex;
	max-width: 100%;
	min-height: auto;
	width: 100%;
```

File: src/resources/postcss/help-hub.pcss [hunk 5]
```diff
@@ -126,6 +116,7 @@ ul.tec-help-list__list-expanded {
 	#tec-settings__support-hub-iframe {
 		height: 750px;
 		width: calc(100% - var(--tec-spacer-3) * 2);
+		margin: 0 var(--tec-spacer-3);
```
Comment (codingmusician) [inline src/resources/postcss/help-hub.pcss:119]:
```suggestion
		height: 750px;
		margin: 0 var(--tec-spacer-3);
		width: calc(100% - var(--tec-spacer-3) * 2);
```

File: src/resources/postcss/help-hub.pcss [hunk 6]
```diff
@@ -137,97 +128,62 @@ ul.tec-help-list__list-expanded {
...
-
-	.tribe_events_page_tec-events-settings {
-		.tec-settings-form {
-			max-width: calc(100% - var(--tec-spacer-7) * 2);
-			width: calc(100% - var(--tec-spacer-7) * 2);
-		}
+	.tribe_events_page_tec-events-settings .tec-settings-form {
+		max-width: initial;
+		width: auto;
+		flex-grow: 1;
```
Comment (codingmusician) [inline src/resources/postcss/help-hub.pcss:154]:
```suggestion
		flex-grow: 1;
		max-width: initial;
		width: auto;
```

**Review by codingmusician — CHANGES_REQUESTED** [review]
Some CSS ordering issues I found.

File: src/admin-views/help-hub.php [hunk 7]
```diff
@@ -36,17 +36,24 @@ class="tribe-events-admin-title__logo"
 		/>
 		Help
 	</h1>
-	<dialog id="tec-settings-nav-modal" class="tec-settings-form__modal">
+	<dialog id="tec-settings-nav-modal" class="tec-settings-form__modal" aria-labelledby="tec-settings-nav-modal-title" aria-modal="true" role="dialog">
```
Comment (Camwyn) [inline src/admin-views/help-hub.php:39]: `aria-labelledby` or `aria-describedby`? Iffy here, there's not a lot of content. I'll leave that up to you.

**Review by codingmusician — APPROVED** [review]

**Review by Camwyn — APPROVED** [review]

**Pull request merged**